lib/device.js

```diff
@@ -14,6 +14,10 @@ function _isChrome(ua) {
   return CHROME_REGEX.test(ua || UA);
 }
 
+function _isOldSamsungBrowserOrSamsungWebview(ua) {
```

Comment:
I like this a little more, because it describes the case it is checking for. If I see:

```
return /SamsungBrowser/.test(ua) || !_isChrome(ua) && ua.indexOf('Samsung') > -1;
```

I won't remember why we have that weird logic in the future.